projects/1.20/assets/create-sifting/createsifter/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "block.createsifter.brass_sifter": "黄铜筛子",
+  "block.createsifter.crushed_end_stone": "粉碎末地石",
+  "block.createsifter.dust": "尘土",
+  "block.createsifter.sifter": "筛子",
+  "create.createsifter.recipe.sifting.minimumspeed": "%1$s RPM",
+  "create.recipe.sifting": "筛选配方",
+  "createsifter.ponder.sifter.header": "方块筛",
+  "createsifter.ponder.sifter.text_1": "筛子通过筛来处理物品",
+  "createsifter.ponder.sifter.text_2": "它们可以使用齿轮从侧面提供动力",
+  "createsifter.ponder.sifter.text_3": "从顶部放入物品",
+  "createsifter.ponder.sifter.text_4": "一段时间后，可以通过右击获得产物",
+  "createsifter.ponder.sifter.text_5": "输出也可以通过自动化提取",
+  "createsifter.recipe.sifting": "筛选配方",
+  "createsifter.recipe.sifting.brass_required": "需要黄铜筛子",
+  "createsifter.recipe.sifting.waterlogged": "浸满水",

```

```suggestion
  "createsifter.recipe.sifting.waterlogged": "含水的",
```
`Waterlogging` 的原版译名为`含水`
译文应以具体实际游戏内容进行翻译